ci/ketch-ci.sh

```diff
@@ -131,12 +131,12 @@ ensure_resource 'ketch-controller-manager' 1
 
 if [ "$RESOURCE_CREATION" = true ] ; then
     # validate addtional required params
-    if [ -z "$POOL" ]; then usage "Pool name required"; fi;
+    if [ -z "$POOL" ]; then usage "Framework name required"; fi;
     if [ -z "$INGRESS_ENDPOINT" ]; then usage "Ingress endpoint required"; fi;
 
-    # Add a pool with ingress Traefik (default), replace ingress endpoint address by your ingress IP address
-    echo "creating pool for deployment ..."
-    POOL_CMD="ketch pool add ${POOL} --ingress-service-endpoint ${INGRESS_ENDPOINT}" 
+    # Add a framework with ingress Traefik (default), replace ingress endpoint address by your ingress IP address
+    echo "creating framework for deployment ..."
+    POOL_CMD="ketch framework add ${POOL} --ingress-service-endpoint ${INGRESS_ENDPOINT}"
```

Comment:
s/POOL/FRAMEWORK/

Reply:
I retained "pool" here. We should probably leave `pool` and `app create` here until we cut another release from the changes in this PR and other recent. I think - I'm not 100% sure that's the best way to handle it.

Reply:
Nevermind me. Sounds like we'll create a release in short order which will legitimize `framework` in this script.